ecommerce/colaraz_features/api/helpers.py

```diff
@@ -0,0 +1,61 @@
+"""
+Helpers for colaraz API.
+"""
+from django.conf import settings
+from django.contrib.sites.models import Site
+
+from oscar.core.loading import get_model
+
+from ecommerce.core.models import SiteConfiguration
+from ecommerce.theming.models import SiteTheme
+
+Partner = get_model('partner', 'Partner')
+
+def get_or_create_site(site_url):
+    site, _ = Site.objects.get_or_create(
+        domain=site_url,
+        name=site_url,
+    )
+    return site
+
+
+def get_partner(site_partner):
+    return Partner.objects.filter(
+        short_code=site_partner
+    ).first()
+
+
+def create_site_theme(site, theme):
+    if settings.DEFAULT_SITE_THEME != theme:
+        SiteTheme.objects.update_or_create(
+            site=site,
+            defaults={
+                'theme_dir_name': theme
+            }
+        )
+
+
+def create_site_configurations(
+    site,
+    partner,
+    lms_site_url,
```

Comment:
```suggestion
    lms_url_root,
```

Just for consistency. Otherwise `lms_site_url` is also fine.

Reply:
`lms_url_root` is being used in for base site e.g: `courses.colaraz.com`, while the logic behind `lms_site_url` related to multisite e.g: `polymath.courses.colaraz.com and insigniabizedu.courses.colaraz.com`